Misc/NEWS.d/next/Core and Builtins/2022-05-02-20-19-06.gh-issue-86199.IZbF0m.rst

```diff
@@ -1,6 +1,6 @@
-`PyObject_Call` now guarantees any kwargs `dict` passed is shallow-copied
-before being passed to the callee, matching the documented equivalence with
-`callable(*args, **kwargs)`, fixing #86795.     When a function call's sole source of
-keyword arguments is a single `**`     unpacked `dict`, no copy is performed
-when the callable uses the vectorcall     protocol and no copy is needed to
-protect the caller's `dict` from mutation.
+:c:func:`PyObject_Call` now guarantees any kwargs :class:`dict` passed is
+shallow-copied before being passed to the callee, matching the documented
+equivalence with ``callable(*args, **kwargs)``.
+When a function call's sole source of keyword arguments is a single ``**``
+unpacked :class:`dict`, no copy is performed when the callable uses
+``vectorcall``, which does not expose the :class:`dict` to the callee.
```

Comment:
```suggestion
:ref:`vectorcall <vectorcall>`, which does not expose the dict to the callee.
```

(Regarding dict markup: it is often enough to provide a link for the first reference. Your call.)